src/index.js

```diff
@@ -136,6 +138,14 @@ const { DEFAULT_ENV, getCliEnv } = require('@adobe/aio-lib-env')
  * @property {string} name the role name
  */
 
+/**
+ * @typedef {object} OauthS2SIntegrationDetails
+ * @property {string} name Name
+ * @property {string} description Description
+ * @property {string} [templateId] - templateId
+ * @property {object} [services] - services
```

Comment:
I believe this is an Array of objects? we should define what Object it is (see SubscribeToServices schema in swagger)
`{Array<SubscribeToServices}` then define what `SubscribeToServices` object is.

Reply:
Yes good callout, [services] is an array of `SubscribeToServices`. I have just pushed changes to accommodate this.